tests/baseline/2.0.x/test_hash_lib.json

```diff
@@ -0,0 +1 @@
+{"test_pytest_mpl.test_hash_succeeds": "cd01b7a39330033b18d54b507635236214cae5e24f9e09d84b56462a9ac81052","test_pytest_mpl.test_hash_fails": "FAIL"}
```

Comment:
I think by default we should probably add a bit of whitespace to have one hash per line, in case any hashes are changed and there are 100 figures!